app/views/admin/facility_groups/_form.html.erb

```diff
@@ -19,9 +19,23 @@
         <% end %>
       <% end %>
 
-      <%= form.select(:protocol_id, protocols.order(:name).map { |protocol| [protocol.name, protocol.id]} ) %>
+      <div class="mt-5 mb-5">
+        <h3>Diabetes enabled?</h3>
+        <div class="alert alert-primary mt-3" role="alert">
+          This will enable Diabetes Management for all associated facilities in this group.
+        </div>
+        <%= form.check_box :enable_diabetes_management, {id: :facility_group_enable_diabetes_management, checked: facility_group.diabetes_enabled?, onclick: "confirmDisableDM(this)"}, true, false %>
+      </div>
+
+      <%= form.select(:protocol_id, protocols.order(:name).map { |protocol| [protocol.name, protocol.id] }) %>
 
       <%= form.primary("Save facility group") %>
     <% end %>
   </div>
 </div>
+<script>
+  function confirmDisableDM(el) {
+    if(!el.checked)
+        confirm("Are you sure? This will disable diabetes management for all associated facilities.")
+  }
```

Comment:
We should only fire this confirmation if `facility_group.diabetes_enabled?`, right? If only a few facilities have DM enabled, the box will be unchecked on the edit screen, and when I save, I shouldn't see any confirmation because we won't change the DM status of any facilities.

Reply:
We might be ditching this JS in favor of not allowing to disable DM at all. Letting Vineet and Praveen discuss it until then. Regardless, this fix has been made even if we keep it.